### DISCUSSION TITLE
Thank you for making and working on Friction!

### DISCUSSION THREAD

**Discussion by Robert-W-K** [friction2d/friction#374] [General]
Also, thank you for modeling a lot of the controls after inkscape. it is a huge relief to find an svg animation program that I can roll right into without too much fuss. I'm hoping I'll be able to accomplish my five second channel identification video with it; I am optimistic and this seems super promising!

**Comment by AlmostMikhail** [comment]
Yeah, I even tried glaxnimate and it's didn't stand up to the performance of friction. It crashed sometimes and the playback was super slow on my low performance laptop. 

 I also tried paid expression Animator but it didn't stand up to the ux/ui speed of friction. Friction just is much easier than expressive animator. I just got frustrated but it didn't have all friction features and shortcuts.

**Comment by nuxttux** [comment]
Hi - I came here to give thanks as well. I'll see how I can support the development. 
Thank you for making and maintaining Friction.

**Comment by donn-xx** [comment]
Wonderful project! Well done and thanks :1st_place_medal:
